src/transformState.js

```diff
@@ -5,7 +5,25 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  const tempState = state;
```

Comment:
you don't need to create extra variable for state because you still work with initial obj ("state")